.azure-pipelines/hpu-tests.yml

```diff
@@ -28,5 +28,21 @@ jobs:
 
     steps:
     - bash: |
+        apt-get install hwinfo
         hwinfo --short
       displayName: 'Instance HW info'
+
+    - bash: |
+        pip install . --requirement requirements/test.txt
+      displayName: 'Install dependencies'
+
+    - bash: |
+         python ".azure-pipelines/run_hpu_tests.py"
```

Comment:
```suggestion
         python .azure-pipelines/run_hpu_tests.py
```

Reply:
I tested with command  python .azure-pipelines/run_hpu_tests.py and does not work as . is considered as regex. Hence, would like to keep it as it is